test/scripts/prepare_git_server.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
:+1:  I would love to see us standardize to using this shebang invocation for all bash WDYT?